docs/sr/week01/01-2.md

```diff
@@ -0,0 +1,158 @@
+---
+lang: sr
+lang-ref: ch.01-2
+lecturer: Yann LeCun
+title: Evolucija i primene konvolucionih neuronskih mreža i zašto duboko učenje?
```

Comment:
The first 'i' may be better be represented by a comma.

Reply:
Agree, updated. Do you see anything else that needs to be changed?